### PR TITLE
#27285 Announcement UI/UX

### LINKED ISSUE #27285: Announcements Menu UI/UX Refinement
### Parent Issue

#25733 

### Task

1. Ensure that the background code is adjusted to enhance the overall visual appeal, keeping it consistently white and avoiding the use of the user variable color.
2. Correct typographical errors in the resources section.
3. Align icons with the title in the resources section to ensure a cohesive visual presentation.
4. Implement the old-style welcome message.
5. ~Ensure that the separator line color matches the color of the right button in the top bar for a consistent visual appearance.~ Remove the separator line between the notifications and announcements buttons.
6. Adjust the padding for the notifications menu.
7. Increase the width of the user menu.
8. Optimize the titles of APIs and services to be less wide. (Based on Figma)
9. Add underlines to the announcements menu Show All button.
10. Increase the height of the announcements section slightly.  (Based on Figma)

### Proposed Objective

Same as Parent Issue

### Proposed Priority

Priority 3 - Average

### Acceptance Criteria

_No response_

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

_No response_

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by manuelrojas**
### Proposed Changes

Adding UI/UX refinement based for the Welcome Page, Announcements Menu and Notifications Menu.

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots

image

image

image

image

image

### CODE REVIEW

**Review by zJaaal — APPROVED** [review]

**Review by oidacra — APPROVED** [review]

**Review by hmoreras — APPROVED** [review]

**Review by KevinDavilaDotCMS — APPROVED** [review]

File: core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-toolbar-user/dot-toolbar-user.component.scss [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 p-avatar {
     cursor: pointer;
-    padding: $spacing-2 0;
+    padding: 0.83rem 0;
```
Comment (rjvelazco) [inline core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-toolbar-user/dot-toolbar-user.component.scss:10]: We should use the `$spacing `variable here

File: core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-toolbar-announcements/dot-toolbar-announcements.component.scss [hunk 2]
```diff
@@ -1,8 +1,8 @@
 @use "variables" as *;
 
 :host {
-    max-height: 37rem;
-    overflow: scroll;
+    max-height: 100vh;
```
Comment (rjvelazco) [inline core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-toolbar-announcements/dot-toolbar-announcements.component.scss:4]: Does this code not create a scroll bar for the page view?

File: core-web/libs/dotcms-scss/angular/dotcms-theme/components/_menu.scss [hunk 3]
```diff
@@ -55,8 +55,8 @@
 .p-menu.p-menu-overlay {
     background: $white;
     border: 0 none;
-    box-shadow: 0 2px 4px -1px $color-palette-black-op-20, 0 4px 5px 0 $color-palette-black-op-10,
-        0 1px 10px 0 $color-palette-black-op-10;
+    box-shadow: $shadow-m;
```
Comment (rjvelazco) [inline core-web/libs/dotcms-scss/angular/dotcms-theme/components/_menu.scss:58]: Nice!

**Review by rjvelazco — CHANGES_REQUESTED** [review]

File: core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-toolbar-user/dot-toolbar-user.component.scss (hunk 1 above)
Comment (manuelrojas) [reply core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-toolbar-user/dot-toolbar-user.component.scss]: @rjvelazco Fixed thanks!

**Review by rjvelazco — APPROVED** [review]

**Review by fmontes — APPROVED** [review]

**Pull request merged**